docs/metrics/bertscore.md

```diff
@@ -0,0 +1,122 @@
+
+#### BERT Recall
+
+$$
+R_\text{BERT} = \frac{1}{|x|} \sum_{\mathbf{x_i} \in x} \underbrace{\max_{\mathbf{\hat{x}_j} \in \hat{x}}\overbrace{\mathbf{x_i}^\top \mathbf{\hat{x}_j}}^\text{cosine similarity}}_\text{greedy matching}
+$$
+
+Once again, the BERT recall is conceptually similar to the [recall formula](recall.md). Note that we flip $\hat{x}$ with $x$ when calculating recall.
+
+#### BERT F1-Score
+
+$$
+F_\text{BERT} = 2 \times \frac{P_\text{BERT} \times R_\text{BERT}}{P_\text{BERT} + R_\text{BERT}}
+$$
+
+The formula is the same as the [F1-score formula](f1-score.md), replacing the precision and recall components with BERT precision and recall.
+
+In a more advanced implementation of BERTScore, extra steps are taken to finetune the metric. These include:
+
+1. Applying an "importance factor" to rare words so that the score weighs keywords moreso than words like "it", "as", and "the".
+2. Rescaling the score such that it lies between 0 and 1 in practical use cases. Although the score already lies between 0 and 1 in theory, it has been observed to lie between a more limited range in practice.
+
+[![Bert Computation](../assets/images/bert_computation.png)](https://arxiv.org/pdf/1810.04805.pdf)
+
+### Interpretation
+BERTScore (Precision, Recall, F1) scores lie between the range of 0 and 1, with 0 representing no semantic similarity, and 1 representing a perfect semantic match between candidate and reference texts. However, interpreting the metric is completely subjective based on your task. On some tasks, a BERT-F1 of 0.9 may be excellent, whereas a BERT-F1 of 0.8 may be excellent for another. Generally speaking, a higher BERTScore is desirable.
+
+## Example
+To showcase the value of BERTScore, lets consider the following candidate and reference texts:
+
+??? example "Semantically Similar Texts"
+
+    | Candidate Text | Reference Text |
+    | --- | --- |
+    | The sun set behind the mountains, casting a warm, orange glow across the horizon. | As the mountains obscured the sun, a warm, orange glow painted the horizon. |
+    | She sipped her coffee and gazed out of the window, lost in thought on a rainy afternoon. | Lost in thought on a rainy afternoon, she sipped her coffee and stared out of the window. |
+    | The adventurous explorer trekked through the dense jungle, searching for hidden treasures. | In search of hidden treasures, the intrepid explorer ventured through the dense jungle. |
+    | Laughter echoed through the park as children played on the swings and slides. | Children's laughter filled the park as they enjoyed the swings and slides. |
+    | The old bookstore was filled with the scent of well-worn pages, a haven for book lovers. | A haven for book lovers, the old bookstore exuded the fragrance of well-read pages. |
+
+    Using the following code:
+
+    ```python
+    from bert_score import score
```

Comment:
Should we mention/link somewhere what this library is?  There are also different models that they can use that I think it's worth mentioning.

Reply:
I see that you added `!pip install bert-score`. 

It would be nice to separate that out from the example and add a section under ## implementation header for Python implementation and talk about the library itself and add a link to the source.  I believe there are other libraries like hugging face and pytorch that might be worth mentioning too.